prow/config/config.yaml

```diff
@@ -48,6 +48,36 @@ tide:
     # Use branch protection options to define required and optional contexts
     from-branch-protection: true
 
+branch-protection:
+  orgs:
+    metal3-io:
+      # Require "always_run: true" jobs to pass before merging.
+      # To turn this off for a given job, set "optional: true"
+      # in the job definition.
+      protect: true
+      repos:
+        baremetal-operator:
+          # Use this to specify that a status coming from outside of prow is
+          # required.  We use this to require functional jobs running in
+          # jenkins are required, for example.
+          required_status_checks:
+            contexts: ["test-integration"]
+        cluster-api-provider-metal3:
+          required_status_checks:
+            contexts: ["test-v1a4-integration"]
```

Comment:
Can we specify per branch ? for the release-0.3 branch it would be test-integration. In this current shape, it might prevent us from merging any PR on that branch.

Reply:
Thanks for catching this!  I'll figure how how to do branch settings and post a revision.

Reply:
done.  Take a look and let me know if I've set the right policy.  I have CAPM3 set for `test-integration` except on master it's `test-v1a4-integration`.  I can tweak further if needed.